### DISCUSSION TITLE
Can I move a dashboard between projects?

### DISCUSSION THREAD

**Discussion by risafangs** [lightdash/lightdash#3408] [Q&A] [answered]
I generally test and build my dashboards on development branch (and project), then once the code is merged to production, find that I need to re-build the dashboard. Is there a way to move it to the production project when done?

**Comment by TuringLovesDeathMetal** [comment, marked as answer]
There is a way - but it’s not pretty.
We can use the Lightdash API directly, we can basically scrape all the charts from the dev project and recreate them in the prod project

We have an open issue for being able to move things between projects more easily here - would love to hear your thoughts on it! 😊 : https://github.com/lightdash/lightdash/issues/3014

What's the reason for building dashboards + charts in development first? Is it to test them out? Or is it because you're waiting for the data to get into production? Or is it some other reason?

**Reply by risafangs** [reply]
You basically nailed the reason - the data is pretty new/fast-changing/spotty and I am testing first. Generally speaking, since production is tied to the main/master branch, I would probably be testing anything new on dev branches anyway, so I imagine this would continue to be an issue with in-development models even after the data is more established.

**Reply by TuringLovesDeathMetal** [reply]
cool - makes sense. Testing first seems to be the most common use case for people (which is good to know when we consider how we'll build this 👀 ). 

for now though, it's only available through this hacky API approach. If you emoji-vote on the issue I posted above, I'm _pretty sure_ you'll get updated on its progress!
